[TypeScript] Index signature is missing in type 'DefaultAddOptions'

Since upgrading `typescript@3.5.1` I get this compile error:
```
node_modules/p-queue/index.d.ts:81:54 - error TS2344: Type 'DefaultAddOptions' does not satisfy the constraint 'QueueAddOptions'.
  Index signature is missing in type 'DefaultAddOptions'.

81  EnqueueOptionsType extends PQueue.QueueAddOptions = PQueue.DefaultAddOptions
                                                        ~~~~~~~~~~~~~~~~~~~~~~~~
```
My `tsconfig.json` looks like:
```json
{
	"compilerOptions": {
		"lib": ["esnext"],
		"module": "commonjs",
		"target": "es5"
	}
}
```

--- a/package.json
+++ b/package.json
@@ -59,7 +59,7 @@
 		"random-int": "^2.0.0",
 		"time-span": "^3.1.0",
 		"ts-node": "^8.0.3",
-		"typescript": "^3.4.3",
+		"typescript": "~3.5.1",
 		"xo": "^0.24.0"
 	},
 	"ava": {

--- a/source/options.ts
+++ b/source/options.ts
@@ -71,4 +71,6 @@ export interface DefaultAddOptions {
 	@default 0
 	*/
 	readonly priority?: number;
+
+	readonly [key: string]: unknown;
 }

--- a/source/options.ts
+++ b/source/options.ts
@@ -64,13 +64,11 @@ export interface Options<QueueType extends Queue<QueueOptions>, QueueOptions ext
 	throwOnTimeout?: boolean;
 }
 
-export interface DefaultAddOptions {
+export interface DefaultAddOptions extends QueueAddOptions {
 	/**
 	Priority of operation. Operations with greater priority will be scheduled first.
 
 	@default 0
 	*/
 	readonly priority?: number;
-
-	readonly [key: string]: unknown;
 }

